src/main/java/com/ppkwu/ppkwu/services/SampleService.java

```diff
@@ -12,6 +12,14 @@
 
     public String reverse(String stringToReverse, String reverseToString) {
         String ReVeRsE = "";
+        char temp;
+
+        for (float inputStringLength = stringToReverse.length() - 1; inputStringLength >= 0; inputStringLength--) {
```

Comment:
Typ danych float w tej pętli for może być nieszczęśliwy :) Prawdpodobnie int byłby lepszy. Mógłbyś to zmienić?